KubeArmor/core/unorchestratedUpdates.go

```diff
@@ -436,6 +436,26 @@ func (dm *KubeArmorDaemon) ParseAndUpdateContainerSecurityPolicy(event tp.K8sKub
 		}
 	}
 
```

Comment:
please add a comment specifying why are we doing this operation.

Reply:
What do you think of the comments below?

```
	// When the container name (value of kubearmor.io/container.name) specified in KubeArmorPolicy is changed, it is necessary to update the policy for the container with that name.
	for idx, endPoint := range dm.EndPoints {
		for policyIndex, policy := range endPoint.SecurityPolicies {
			if policy.Metadata["namespaceName"] == secPolicy.Metadata["namespaceName"] && policy.Metadata["policyName"] == secPolicy.Metadata["policyName"] && endPoint.EndPointName != containername {
```